templates/tasks.cron.erb

```diff
@@ -0,0 +1,6 @@
+SHELL=/bin/sh
+RAILS_ENV=production
+FOREMAN_HOME=/usr/share/foreman
+
+# Clean up expired tasks from the database
+<%= @cron_line %>    foreman    /usr/sbin/foreman-rake foreman_tasks:cleanup >>/var/log/foreman/cron.log 2>&1
```

Comment:
The foreman username is also available as the variable ```foreman::user```.

Reply:
fixed